examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+"""
+==================================================================
+Demonstrate morphing subject source volume to fsaverage using dipy
+==================================================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose dipy (http://nipy.org/dipy/) will be
+used in order to perform the necessary transforms to a fsaverage serving as
+reference space. The example uses parts of the MNE example lcmv beamformer
+pipeline (examples/inverse/plot_lcmv_beamformer_volume.py).
```

Comment:
Should be something like:
```
:ref:`LCMV beamformer pipeline <sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py>`
```
Try it and see if it renders properly with `make html_dev-noplot`